Exercise 1/Exercise1.rst

```diff
@@ -1,5 +1,3 @@
-Look at the file titled titled "Match This Output.png" in this directory.
-
-Use this space to create some RST that matches the screenshot.
-
-First... delete all this text!
+New Article
+###########
```

Comment:
Stylistically, you'll need a blank line after header markings (such as `#######`)